presto-product-tests/src/main/resources/sql-tests/testcases/window_functions/windowSpecificationAABcp.sql

```diff
@@ -0,0 +1,8 @@
+-- database: presto; groups: window;
```

Comment:
Comments added.
I propose merging the .sql and .results files together for all of the convention-based tests in here in a separate change. Thoughts?